README.md

```diff
@@ -129,4 +170,12 @@ If you think you have discovered a security issue in this code please email [dis
 For non-security related bugs and feature requests please [raise an issue](https://github.com/alphagov/verify-service-provider/issues/new) in the GitHub issue tracker.
 
 ## Licensing
-[MIT License](https://github.com/alphagov/verify-service-provider/blob/master/LICENSE)
+[MIT License][mit-license]
+
+[key-rotation]: https://www.docs.verify.service.gov.uk/maintain-your-connection/rotate-keys/
+[vsp-get-started]: https://www.docs.verify.service.gov.uk/get-started-with-vsp/
+[vsp-api]: https://github.com/alphagov/verify-service-provider/blob/master/architecture-decisions/verify-service-provider-api.swagger.yml
```

Comment:
This is going to be a little wrong, but we can fix it later.

Reply:
Do you mean the VSP API reference location? Yeah, we should remember to include the README updates when we pick up the bit of work...